pytest_fixtures.py

```diff
@@ -0,0 +1,62 @@
+import pytest
+
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+
+
+@pytest.fixture(scope='module')
+def db_url():
+    """ Database URL used in sqlalchemy.create_engine
+
+    Override this in your test to provide your desired url.
+    For valid urls see: http://docs.sqlalchemy.org/en/latest/core/engines.html
+    Defaults to SQLite memory database.
+    """
+    return 'sqlite://'
```

Comment:
It would be nice to pull this out of pytest options, as well as having the option of overriding the fixture if you wanted. @juliotrigo has a nice pattern for this in the conftest of https://github.com/Overseas-Student-Living/sqlalchemy-filters/pull/1